docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,111 @@
+
+  ![!](../../_images/PMM_access_edit_labels.png)
+
+Furthermore, in our continued effort  to enhance user experience, we have redesigned the following pages to make them more user-friendly and intuitive:
+
+- Select service type page
+- Add service page
+
+
+#### Connecting Services and Nodes 
+
+Starting with PMM 2.40.0, you can click on the link in the **Node Name** column to view the node on which a specific service is running and analyze how node-level resource utilization impacts the performance of those services.
+
+Before introducing this feature, locating the running services on a node was cumbersome. However, with this new feature, you can effortlessly access a list of services running on a specific node and identify the node name where the service is being utilized. Additionally, the filters implemented make navigation a lot simpler.
+
+  ![!](../../_images/PMM_Inventory_service_node_relation.png)
+
+Furthermore, you can also see the service running on that specific node when you click on the link in the **Services** column.
+
+  ![!](../../_images/PMM_Inventory_node_service_relation.png)
+
+
+
+#### Cluster view for Inventory [Technical Preview]
+
+Understanding the structure of your inventory is crucial. With the release of PMM 2.40.0, we've introduced an experimental feature that categorizes all your Services by Cluster, making it easier for you to understand your inventory. By using the **Organize by Clusters** toggle, you can view a group of services as a single cluster. PMM utilizes the cluster label to display services that belong to the same cluster.
+
+  ![!image](../../_images/PMM_Inventory_cluster_view.png)
+
+For detailed information on this feature, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-inventory.html#cluster-view)
+ 
+
+### MongoDB Backup Monitoring 
+
+With this release, we are shipping the first version of a the **Backup failed** alert template which notifies of any failed MongoDB backups. This uses a new, dedicated metric, `pmm_managed_backups_artifacts`, for checking the state of backup artifacts. 
+
+This new template is currently in Technical Preview and we are keen to get your feedback around this change before we move it to General Availability.
+
+  ![!image](../_images/Backup_Failed_Template.png) 
+
+For information on working with this new template, see the [Percona Alerting](../get-started/alerting.md) topic.
+
+
+### Components Upgrade
+
+VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHANGELOG.html#v1934).
+
+
+## New Features
+
+- [PMM-9374](https://jira.percona.com/browse/PMM-9374) - Starting with PMM 2.40.0, you can now use an external VictoriaMetrics database for monitoring in PMM. This provides multiple benefits, including scalability, resource isolation, reduced load on the PMM server,  customization, etc.
```

Comment:
this needs to be marked as technical preview and be at the bottom of the section

Reply:
Done

Reply:
Done.